metadata-ingestion/src/datahub/ingestion/source/superset.py

```diff
@@ -232,6 +236,7 @@ def __init__(self, ctx: PipelineContext, config: SupersetConfig):
                 graph=self.ctx.graph,
             )
         self.session = self.login()
+        self.owners_dict = self.build_preset_owner_dict()
```

Comment:
What are the key and value in this dictionary? You might want to consider using a field or function name that clearly indicates the purpose of the key and value in the resulting dictionary.